core/sodasql/cli/cli.py

```diff
@@ -62,6 +67,19 @@ def create(warehouse_type: str,
 
     WAREHOUSE_TYPE is one of {postgres, snowflake, redshift, bigquery, athena}
     """
+
+    soda_telemetry.set_attribute('cli_command_name', 'create')
+    soda_telemetry.set_attribute('datasource_type', warehouse_type)
+
+    span_setup_function_args(
+        locals(),
```

Comment:
This is a bit risky code and it might raise eyebrows because we end up sending password (which is part of locals) to the span_setup_function. 

I'd rather make the args arg (for span_setup_function) more explicit. 

WDYT?

Reply:
agreed, changed to explicit dicts. This was unnecessarily complex, doing it explicitly simplified the logic a lot as well.